src/client/App.js

```diff
@@ -27,6 +28,7 @@ function App() {
         <SignIn exact path="/sign-in" />
         <SignUp exact path="/sign-up" />
         <ResetPassword exact path="/reset-password" />
+        <DeleteBoardModal exact path="/deleteBoard" />
```

Comment:
We need to get the boardId from the parameter.
So /deleteBoard/:boardId
then
const { boardId } = useParams();

Reply:
Also, It must be in an authenticated route

Reply:
I willbe making the changes soon

Reply:
> Also, It must be in an authenticated route

Delete board modal is gonna be nested in the ownerBoardPage and the profilePage. so it doesn't need any authenticated route right.

Reply:
Ok